### DISCUSSION TITLE
another tool than vcf2maf?

### DISCUSSION THREAD

**Discussion by complexgenome** [PoisonAlien/maftools#961] [General]
Hi there, 

is there another tool to make MAF from VCF files?
https://github.com/mskcc/vcf2maf

**Comment by PoisonAlien** [comment]
Hi,
You can try [Funcotator](https://gatk.broadinstitute.org/hc/en-us/articles/360035889931-Funcotator-Information-and-Tutorial). It works nicely with vcf files and outputs annotated maf.

**Comment by complexgenome** [comment]
Thanks for your reply. Yes, I am exploring gatk's functator.
